api/src/database/index.ts

```diff
@@ -143,6 +143,19 @@ export function getDatabase(): Knex {
 		};
 	}
 
+	if (client === 'oracledb') {
+		poolConfig.afterCreate = async (conn: any, callback: any) => {
+			logger.trace('Setting OracleDB NLS_DATE_FORMAT and NLS_TIMESTAMP_FORMAT');
+
+			// enforce proper ISO standard date as default
+			await conn.executeAsync('ALTER SESSION SET NLS_TIMESTAMP_FORMAT = \'YY-MM-DD"T"HH24:MI:SS\'');
```

Comment:
Should it be `YYYY` instead of `YY` for both formats? 

How about the `NLS_TIMESTAMP_TZ_FORMAT`?

Reply:
> Should it be YYYY instead of YY for both formats?

Either seem to work but in the interest of non ambiguiigty updating it 👍 

> How about the NLS_TIMESTAMP_TZ_FORMAT?

This does not seem to be necessary as `NLS_TIMESTAMP_FORMAT` [^1] covers both `TIMESTAMP` and `TIMESTAMP WITH LOCAL TIME ZONE` which are the only 2 we seem to user

---
[^1] : https://docs.oracle.com/en/database/oracle/oracle-database/19/nlspg/datetime-data-types-and-time-zone-support.html#GUID-3A1B7AC6-2EDB-4DDC-9C9D-223D4C72AC74